apps/dash-uber-rides-demo/README.md

```diff
@@ -0,0 +1,48 @@
+# Dash Uber Rides Demo
+
+## About this app
+
+This is a demo of the Dash interactive Python framework developed by [Plotly](https://plot.ly/).
+
+This demo is based off of data provided by [FiveThirtyEight](https://github.com/fivethirtyeight/uber-tlc-foil-response/tree/master/uber-trip-data) and highlights Uber trips taken in New York City from April 2014 to September 2014.
+
+
+## How to run this app
+
+(The following instructions apply to Windows command line.)
+
+To run this app first clone repository and then open a terminal to the app folder.
+
+```
+git clone https://github.com/plotly/dash-sample-apps.git
+cd apps/dash-uber-rides-demo
+```
+
+Create and activate a new virtual environment (recommended) by running
+the following:
+
+```
+virtualenv venv 
+/venv/scripts/activate
```

Comment:
slight change, but you can add the linux version like so
# Windows
dash-tsne-venv\Scripts\activate
# Or Linux
source dash-tsne-venv/bin/activate